components/automate-pg-gateway/habitat/config/haproxy.conf

```diff
@@ -47,17 +47,18 @@ frontend postgresql
 backend automate_postgresql
 {{~#if cfg.service.external_postgresql.enable }}
     balance roundrobin
-    option tcp-check
+    option httpchk GET /
+    option redispatch
     default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
     {{~#each cfg.service.parsed_nodes as |node|}}
         {{~#if node.is_domain}}
             {{~#if ../cfg.resolvers.nameservers }}
     server-template {{node.address}} 8 {{node.address}}:{{node.port}} check resolvers pgdns init-addr none  resolve-prefer ipv4
             {{else}}
-    server {{node.address}} {{node.address}}:{{node.port}} check
+    server {{node.address}} {{node.address}}:{{node.port}} maxconn 350 check port 6432
```

Comment:
if we consider this value -> maxconn 350
earlier this is the part of backend ha proxy and its value is 350. And there are 3 HAproxy at backend.
Now the pg-gateway on each FE will have this value, does we need to increase the number of PG connection on the backend package ?

Reply:
increased default max_connection to 1500